pkg/ansible/runner/runner.go

```diff
@@ -82,19 +85,28 @@ func NewFromWatches(path string) (map[schema.GroupVersionKind]Runner, error) {
 			Version: w.Version,
 			Kind:    w.Kind,
 		}
+		var reconcilePeriod time.Duration
+		if w.ReconcilePeriod != "" {
+			d, err := time.ParseDuration(w.ReconcilePeriod)
+			if err != nil {
+				logrus.Warningf("unable to parse duration: %v - %v, setting to default", w.ReconcilePeriod, err)
```

Comment:
If someone tries to set this but screws it up, I think it's better to fail to make sure they know and fix it.